meta-lmp-base/classes/uboot-fitimage.bbclass

```diff
@@ -20,7 +20,7 @@ UBOOT_ITS_SYMLINK ?= "u-boot-${MACHINE}.its"
 
 # fitImage Hash Algo
 FIT_HASH_ALG ?= "sha256"
-OPTEE_BINARY ?= "tee-pager.bin"
+OPTEE_BINARY ?= "tee-pager_v2.bin"
```

Comment:
Did you check if optee 3.6 is also generating this v2 binary? IIRC that could end up breaking 3.6 users.

Reply:
Yes, I belive I did. Recheck it this morning.

Reply:
Recheck. Yes, 3.6 generates v2 and the newer versions don't generate v1, that's why this change was backported.